src/sentinel.c

```diff
@@ -3441,6 +3446,215 @@ void addReplySentinelRedisInstance(client *c, sentinelRedisInstance *ri) {
+            robj *o = c->argv[++j];
+            if (getLongLongFromObject(o,&ll) == C_ERR || ll <= 0) {
+                badarg = j;
+                goto badfmt;
+            }
+            sentinel_tilt_period = ll;
+           
+        } else if (!strcasecmp(option,"slave_reconf_timeout") && moreargs > 0) {
+            /* slave_reconf_timeout <milliseconds> */
+            robj *o = c->argv[++j];
+            if (getLongLongFromObject(o,&ll) == C_ERR || ll <= 0) {
+                badarg = j;
+                goto badfmt;
+            }
+            sentinel_slave_reconf_timeout = ll;
+           
+        } else if (!strcasecmp(option,"min_link_reconnect_period") && moreargs > 0) {
+            /* min_link_reconnect_period <milliseconds> */
+            robj *o = c->argv[++j];
+            if (getLongLongFromObject(o,&ll) == C_ERR || ll <= 0) {
+                badarg = j;
+                goto badfmt;
+            }
+            sentinel_min_link_reconnect_period = ll;
+           
+        } else if (!strcasecmp(option,"default_failover_timeout") && moreargs > 0) {
+            /* default_failover_timeout <milliseconds> */
+            robj *o = c->argv[++j];
+            if (getLongLongFromObject(o,&ll) == C_ERR || ll <= 0) {
+                badarg = j;
+                goto badfmt;
+            }
+            sentinel_default_failover_timeout = ll;
+           
+        } else if (!strcasecmp(option,"election_timeout") && moreargs > 0) {
+            /* election_timeout <milliseconds> */
+            robj *o = c->argv[++j];
+            if (getLongLongFromObject(o,&ll) == C_ERR || ll <= 0) {
+                badarg = j;
+                goto badfmt;
+            }
+            sentinel_election_timeout = ll;
+           
+        } else if (!strcasecmp(option,"script_max_runtime") && moreargs > 0) {
+            /* script_max_runtime <milliseconds> */
+            robj *o = c->argv[++j];
+            if (getLongLongFromObject(o,&ll) == C_ERR || ll <= 0) {
+                badarg = j;
+                goto badfmt;
+            }
+            sentinel_script_max_runtime = ll;
+           
+        } else if (!strcasecmp(option,"script_retry_delay") && moreargs > 0) {
+            /* script_retry_delay <milliseconds> */
+            robj *o = c->argv[++j];
+            if (getLongLongFromObject(o,&ll) == C_ERR || ll <= 0) {
+                badarg = j;
+                goto badfmt;
+            }
+            sentinel_script_retry_delay = ll;
+           
+        } else {
+            addReplyErrorFormat(c,"Unknown option or number of arguments for "
+                                  "SENTINEL SET '%s'", option);            
+        }
+
+    }
+
+    addReply(c,shared.ok);
+    return;
+
+badfmt: /* Bad format errors */
+    addReplyErrorFormat(c,"Invalid argument '%s' for SENTINEL SET '%s'",
+        (char*)c->argv[badarg]->ptr,option);
+
+    return;
+}
+
+
+void addReplySentinelDebugInfo(client *c) {
```

Comment:
Consider aligning the parameter names reported in `DEBUG` output with the parameter names accepted when setting a value - I think it's easier to use/understand.

Reply:
Updating codes as follow:   1. all arguments maintain consistency with other arguments that use a hyphen (tilt-trigger etc)
2. sentinel debug command output aligns with parameter name.  
Thanks